### DISCUSSION TITLE
Set Expiry Date Dynamically

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-future#440] [Feature Requests]
> I use Gravity Forms to create posts for a Ride Exchange site on our campus intranet. The form contains a field labeled “Departure Date” which is output in the post content.

> How might I use the field data to set the Expiration upon Post creation?

Original request: https://wordpress.org/support/topic/set-expiry-date-dynamically/

**Comment by andergmartins** [comment]
Related to #364

**Comment by andergmartins** [comment]
We've implemented this for workflows by using custom metadata.
